fuse/all.py

```diff
@@ -170,3 +184,9 @@ def fuse_stop_and_search():
     allegation_df.to_csv(data_file_path("fuse/allegation.csv"), index=False)
     uof_df.to_csv(data_file_path('fuse/use_of_force.csv'), index=False)
     sas_df.to_csv(data_file_path('fuse/stop_and_search.csv'), index=False)
+    post_events_df = fuse_post_events()
+    post_events_df = post_events_df.loc[~post_events_df.agency.isin(event_df.agency.unique())]
```

Comment:
As mentioned above, we should raise an error if there are any agency text in `event_df` that does not appear in POST, which would indicate that we need to make a change in the clean script. Unless there are actually agencies that do not appear in POST. Are there agencies like that @ayyubibrahimi ?

Reply:
I'm on it. All official agencies do appear in POST.